packages/local_auth/local_auth/windows/local_auth_plugin.cpp

```diff
@@ -0,0 +1,187 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+#include "include/local_auth/local_auth_plugin.h"
+
+// This must be included before many other Windows headers.
+#include <UserConsentVerifierInterop.h>
+#include <flutter/encodable_value.h>
+#include <flutter/method_channel.h>
+#include <flutter/plugin_registrar_windows.h>
+#include <flutter/standard_method_codec.h>
+#include <pplawait.h>
+#include <ppltasks.h>
+#include <windows.h>
+#include <winrt/Windows.Foundation.h>
+#include <winrt/Windows.Security.Credentials.UI.h>
```

Comment:
We should clearly document the Windows version requirements in the README of the new package.

Since I'm not familiar with how WinRT linking works: what are the implications of the dependency here on support for older versions of Windows? Will a project that includes `local_auth` completely fail to launch on, e.g., Windows 7, or will it only be a problem if the plugin code is called?

(Pre-Windows-10 support definitely isn't required, I just want to make sure we know what will happen.)